orion/sites/admin/src/achieve/achieve.js

```diff
@@ -2,13 +2,54 @@
 require('./achieve.styl');
 import React from 'react';
 import ReactDOM from 'react-dom';
+import API from '../api.js';
+import { Modal } from '../modals/modal.js';
```

Comment:
same here. You may have to use it later, but it's best to add it in when you need it and keep your code tidy.